themes/beaver/assets/js/tabs.js

```
@@ -0,0 +1,46 @@
+document.querySelectorAll('.js-tab').forEach(el => {
+    const container = el.closest('.js-tabs-container');
+    el.onclick = function (e) {
+        const targetTab = e.target;
+        const panel = document.getElementById(targetTab.getAttribute('aria-controls'));
+        container.querySelectorAll(".tab-panel").forEach(panel => panel.hidden = true);
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)